### PR TITLE
feat(lang): io.http download files to extra_source

### LINKED ISSUE #826: feat(lang): Support download files from internet.
## Description

`download("https://...../data.zip", checksum="...")`

---

**Message from the maintainers**:

Love this enhancement proposal? Give it a 👍. We prioritise the proposals with the most 👍.

### PR DESCRIPTION
**Pull request by kemingy**
Signed-off-by: Keming <kemingyang@tensorchord.ai>

* close #826

### CODE REVIEW

**Review by gaocegege — DISMISSED** [review]

**Comment by kemingy** [comment]
cc @VoVAllen 

- Can this solve your requirement in #826?

File: pkg/lang/ir/interface.go [hunk 1]
```diff
@@ -185,6 +186,22 @@ func Mount(src, dest string) {
 	})
 }
 
+func HTTP(url, checksome, filename string) error {
```
Comment (VoVAllen) [inline pkg/lang/ir/interface.go:189]: checksum?

Comment (kemingy) [reply pkg/lang/ir/interface.go]: typo, fixed

**Review by VoVAllen — APPROVED** [review]
LGTM

**Pull request merged**